src/main/java/org/pih/petl/api/JobExecutionStatus.java

```diff
@@ -0,0 +1,12 @@
+package org.pih.petl.api;
+
+public enum JobExecutionStatus {
+    INITIATED,
+    QUEUED,
+    IN_PROGRESS,
+    FAILED_WILL_RETRY,
+    RETRY_QUEUED,
+    SUCCEEDED,
+    FAILED,
+    ABORTED
+}
```

Comment:
I changed the status into something more structured with this enum, and added some additional statuses than were originally supported.